docs/federation.md

```diff
@@ -173,7 +198,7 @@ public List<Review> reviews(DgsDataFetchingEnvironment dataFetchingEnvironment)
 
 ### Testing a Federated Query
 
-You can always manually test federated queries by running the gateway and your DGS locally. 
+You can always manually test federated queries by running the gateway and your DGS locally.
```

Comment:
I think we should flip this text around and focus on testing *without* running a gateway, so:

1. Write a test
2. Test by sending a federated query directly to the DGS (e.g. using graphiql)
3. Run your gateway locally